### ISSUE TITLE
Other page specific accessibility issues

### ISSUE THREAD

**Issue by struthj** [SmarterApp/AP_ItemSampler#5]
- [x] Other IssuesThe template uses` <nav>, <main>`, and `<footer>`to identify important regions within the page. This is good, overall structure, but the `<nav> `element should not include the main logo and heading. Instead, use the following structure: <header>[main logo and heading`]<nav>`[navigation links]`</nav></header>`. Severity 3

- [x]  Dialogs on this page have the following accessibility issues: //SHIFT+TAB GOES OUT OF MODAL

- The button that opens the dialogs should have aria-haspopup="dialog"to indicate to screen reader users that it opens a dialog. Severity 4
- They trap or maintain keyboard focus when navigating forward with a keyboard. They must also maintain focus when the user navigates backward by pressing Shift + Tab. Otherwise, a keyboard or screen reader user can interact with content that is meant to be hidden or inaccessible. Severity 2

- Each dialog begins with a fourth-level heading. We recommend changing this to a second-level heading. Severity 3

- [x]  The About Test Itemsand Item Detailspages have dialogs with the following accessibility issues:

- The button that opens the dialogs should be marked up as true <button> elements instead of links. 
They should have aria-haspopup="dialog"to indicate to screen reader users that it opens a dialog. Severity 3

- They trap or maintain keyboard focus when navigating forward with a keyboard. They must also maintain focus when the user navigates backward by pressing Shift + Tab. Otherwise, a keyboard or screen reader user can interact with content that is meantto be hidden or inaccessible. Severity 2

- Each dialog begins with a fourth-level heading. We recommend changing this to a second-level heading, and adjusting other heading levels within the dialogs as needed. Severity 3
EDIT changed to h3 as h2 messed up template look
